### PR TITLE
events: Listen for task completion pulse messages

### PR DESCRIPTION
**Pull request by marco-c**

### CODE REVIEW

File: events/code_review_events/workflow.py [hunk 1]
```diff
@@ -257,6 +262,16 @@ def __init__(self, cache_root):
         )
         self.monitoring.register(self.bus)
 
+        # Create pulse listener for unit test failures
+        self.pulse = PulseListener(
```
Comment (La0) [inline events/code_review_events/workflow.py]: Please rename to `pulse_unit_tests` or anything with a suffix, as i'll need `pulse_autoland` pretty soon !

File: events/code_review_events/__init__.py [hunk 2]
```diff
@@ -6,3 +6,4 @@
 QUEUE_MONITORING = "monitoring"
 QUEUE_PHABRICATOR_RESULTS = "results"
 QUEUE_WEB_BUILDS = "builds"
+QUEUE_PULSE = "pulse"
```
Comment (La0) [inline events/code_review_events/__init__.py]: Please rename to `QUEUE_PULSE_UNIT_TESTS`

File: events/code_review_events/workflow.py (hunk 1 above)
Comment (marco-c) [reply events/code_review_events/workflow.py]: You might be able to use the same listener, depending on how you want to implement that.

**Comment by marco-c** [comment]
Given that we don't know yet how the autoland stuff will be implemented (maybe it'll just look for the same pulse message), I'll merge this as-is and we can change the naming when the time comes.

**Pull request merged**